localstripe/resources.py

```diff
@@ -2878,3 +2878,180 @@ def __init__(self, card=None, customer=None, **kwargs):
+
+            if metadata is not None:
+                assert type(metadata) is dict
+            else:
+                metadata = {}
+
+            if statement_descriptor is not None:
+                assert type(statement_descriptor) is str
+
+                if len(statement_descriptor) > 22:
+                    msg = 'Parameter "statement_desciptor" '\
+                        'longer than 22 characters'
+                    raise UserError(400, msg)
+
+            else:
+                statement_descriptor = ''
+
+            if destination is not None:
+                raise UserError(400, 'Custom payout destination not supported')
+            else:
+                destination = f'ba_{random_id(24)}'
+
+            if method is not None:
+                assert type(method) is str
+
+                if method not in ('standard', 'instant'):
+                    raise UserError(400, 'Bad payout method')
+            else:
+                method = 'standard'
+
+            if source_type is not None:
+                assert type(source_type) is str
+
+                if source_type not in ('bank_account',
+                                       'card', 'fpx'):
+                    raise UserError(400, 'Bad payout source_type')
+            else:
+                source_type = 'card'
+
+            # Fake balance has no funds
+            if no_funds is not None:
+                no_funds = try_convert_to_bool(no_funds)
+                assert type(no_funds) is bool
+            else:
+                no_funds = False
+
+            # Fake status, because we cannot
+            # fake a cronable status update
+            if status is not None:
+                assert type(status) is str
+
+                if status not in ('paid', 'pending', 'failed'):
+                    raise UserError(400, 'Bad payout status')
+
+        except AssertionError:
+            raise UserError(400, 'Bad request')
+
+        # TODO
+        # Check that the amount, currency and source_type
+        # is avaliable via the Balance resource implemented in PR#165
+        # For the moment, we accept all payouts as of infinite balance
+        # and fake insufficient funds via the no_funds boolean parameter
+        if no_funds:
+            raise UserError(400, 'Insufficient Funds')
+
+        # All exceptions must be raised before this point.
+        super().__init__()
+
+        # Init defined variables
+        self.amount = amount
+        self.currency = currency
+        self.description = description
+        self.destination = destination
+        self.metadata = metadata
+        self.method = method
+        self.source_type = source_type
+        self.statement_descriptor = statement_descriptor
+
+        # By default expected arrival to two days after creation
+        two_days = 60 * 60 * 24 * 2
+        self.arrival_date = int(time.time() + two_days)
+
+        # As it was manually created
+        self.automatic = False
```

Comment:
At payout creation this property is "True" by default no?
I think I don't understand the comment entirely.

Reply:
Yes, actually its `False`, Stripe doing their thing... https://stripe.com/docs/api/payouts/object#payout_object-automatic we put it to false because '_false if it was requested manually._'